docs/recipes/typescript.md

```diff
@@ -83,3 +83,7 @@ test('an actual test', t => {
 ```
 $ npm test
 ```
+
+## Snapshots
+
+Ava derives the base location for the snapshot fixtures relative to the typescript as if it were standard javascript tests. It determines this from the associated `*.js.map` sourcemap files. Enabling sourcemaps with the typescript compiler can be done in either the [tsconfig](http://www.typescriptlang.org/docs/handbook/tsconfig-json.html) `tsconfig.json#compilerOptions#sourceMap=true` or in the `tsc` cli option `--sourceMap`.
```

Comment:
Rather than this paragraph, update the sample `tsconfig.json` to enable source maps. Honestly it should always be enabled, since it helps AVA show correct stack traces and code snippets when assertions fail.